### DISCUSSION TITLE
How about adding a discord server to boost this development and have a community?

### DISCUSSION THREAD

**Discussion by bokki007** [Shabinder/SpotiFlyer#1823] [Ideas]
I think it is better to have a discord sever for this project!

There are issues with this softaware. If this is a success this will be million dollar success!
https://i.imgur.com/KxDmofJ.jpg

**Comment by UNKNOWTROLLERHASNOTBEENSEEN** [comment]
DO YOU WANT TO JOIN A DISCORD SEVER no issue THERE IS NO RULE ON THIS DISCORD SEVER SO JOIN IF YOU WANT :D 
HERE: https://discord.gg/3cNHrJB4Yy
